quinn-proto/src/crypto.rs

```diff
@@ -390,12 +398,26 @@ pub struct ZeroRttCrypto {
     cipher: &'static aead::Algorithm,
 }
 
-#[derive(Clone)]
 pub struct CryptoContext {
     local: CryptoState,
     remote: CryptoState,
     digest: &'static digest::Algorithm,
     cipher: &'static aead::Algorithm,
+    sealing_key: aead::SealingKey,
+    opening_key: aead::OpeningKey,
+}
+
+impl Clone for CryptoContext {
+    fn clone(&self) -> CryptoContext {
+        Self {
+            sealing_key: aead::SealingKey::new(self.cipher, &self.local.key.clone()).unwrap(),
+            opening_key: aead::OpeningKey::new(self.cipher, &self.remote.key).unwrap(),
+            local: self.local.clone(),
+            remote: self.remote.clone(),
+            digest: self.digest,
+            cipher: self.cipher,
+        }
+    }
```

Comment:
I don't think we actually clone this structure ever, so we might as well remove the impl.